assets/css/home.css

```diff
@@ -0,0 +1,660 @@
+  top: 25px;
+}
+
+.nav-bar li {
+  font-size: 25px;
+  font-weight: bold;
+  color: var(--secondary-white);
+  margin-bottom: 25px;
+}
+
+.nav-bar-ul {
+  flex-direction: column;
+  margin: 50px;
+}
+
+.nav-bar-ul li {
+  list-style: none;
+}
+
+.nav-bar-ul a {
+  color: var(--dark-color);
+  margin-right: 25px;
+  font-size: 30px;
+  font-weight: bold;
+  text-decoration: none;
+}
+
+.social-media {
+  display: none;
+}
+
+article.center-text {
+  text-align: center;
+}
+
+.set-background {
+  background-image: url(../images/bg/mobile_intro_bg.png);
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-position: center;
+  padding-inline: 8vw;
+  margin: 2px 1px;
+}
+
+.Hero {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  padding: calc(min(80px, 10%)) 0;
+  gap: 10px;
+}
+
+h2 {
+  margin-top: 10%;
+  margin-bottom: 0;
+  font-size: 1.6rem;
+  font-weight: 600;
+}
+
+h1 {
+  font-weight: 900;
+  font-size: 2rem;
+  text-transform: uppercase;
+  background-image: url(../images/text-bg.jpg);
+  background-clip: inherit;
+  -webkit-background-clip: text;
+  -webkit-text-fill-color: transparent;
+  display: block;
+  background-size: cover;
+  margin-top: 10px;
+  padding-bottom: 20px;
+}
+
+.title h2 {
+  color: var(--secondary-white);
+  text-align: center;
+  font-size: 1.6rem;
+  font-weight: 400;
+}
+
+.Hero p {
+  display: block;
+  margin-block-start: 1em;
+  margin-block-end: 1em;
+  margin-inline-start: 0;
+  margin-inline-end: 0;
+  border: solid #fff;
+  background-color: #d3d3d347;
+  font-size: 0.9rem;
+  font-weight: 600;
+  line-height: 1.6rem;
+}
+
+h3 {
+  margin-block-start: 1.33em;
+  margin-block-end: 1.33em;
+  margin-inline-start: 0;
+  margin-inline-end: 0;
+}
+
+p.date {
+  font-weight: 900;
+  font-size: 1.5rem;
+}
+
+h4 {
+  display: block;
+  margin-block-start: 1em;
+  margin-block-end: 1em;
+  margin-inline-start: 0;
+  margin-inline-end: 0;
+}
+
+.main-text-color {
+  color: var(--main-color);
+}
+
+.dark-text-color {
+  color: var(--dark-color);
+}
+
+.white-text-color {
+  color: var(--primary-white);
+}
+
+.text-secondary-white {
+  color: var(--secondary-white);
+}
+
+.main-bg-color {
+  background-color: var(--main-color);
+}
+
+.dark-bg-color {
+  background-color: var(--dark-color);
+}
+
+.text-center {
+  text-align: center;
+}
+
+.hide {
+  display: none;
+}
+
+.underline {
+  height: 1px;
+  width: 18%;
+  margin: 1rem auto;
+  background-color: var(--main-color);
+}
+
+.program {
+  display: block;
+}
+
+.program_content {
+  color: #d3d3d3;
+  padding: 1rem;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 1rem;
+  background-image: url(../images/pattern.jpg);
+  background-repeat: repeat;
+}
+
+.program_details {
+  display: flex;
+  flex-direction: column;
+  gap: 10px;
+  width: 100%;
+}
+
+.program_card {
+  display: grid;
+  grid-template-columns: 15% 25% auto;
+  align-items: center;
+  background-color: #ffffff21;
+}
```

Comment:
- [ ] Please adjust the font size and spacing of your program section mobile version to give the content more space and make sure they don't collapse with each other just like in the provided design. Example snippet:

```suggestion
.program_card {
    display: grid;
    grid-template-columns: 15% 25% auto;
    align-items: center;
    background-color: #ffffff21;
    column-gap: 1rem;
    padding: 0.5rem 1rem;
    font-size:12px;
    }
```

Your design| Cindy Shin's design
|---|---|
|<img width="471" alt="Screenshot 2023-06-30 at 7 49 22 AM" src="https://github.com/yusufyusie/Africa-Day-2023-Concert/assets/44798044/62b850e2-9ed6-43d1-a7ce-c02033d27acc">|<img width="471" alt="Screenshot 2023-06-30 at 8 15 52 AM" src="https://github.com/yusufyusie/Africa-Day-2023-Concert/assets/44798044/5c303a74-f7fe-4969-be7e-25dc997225e5">|